### PR TITLE
Handle WalletSelectorAction flag 

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

File: universal-login-wallet/src/ui/react/Login/Login.tsx [hunk 1]
```diff
@@ -45,6 +45,11 @@ const Login = ({location} : LoginProps) => {
           onConnectionClick={onConnectionClick}
           sdk={sdk}
           domains={walletConfig.domains}
+          actions={[
```
Comment (marekkirejczyk) [inline universal-login-wallet/src/ui/react/Login/Login.tsx]: Add default

File: universal-login-react/src/core/models/WalletSelectorAction.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export enum WalletSelectorAction {
```
Comment (marekkirejczyk) [inline universal-login-react/src/core/models/WalletSelectorAction.ts]: Move to commons

File: universal-login-wallet/src/ui/react/Login/Login.tsx (hunk 1 above)
Comment (xitronix) [reply universal-login-wallet/src/ui/react/Login/Login.tsx]: ✔️

File: docs/modeling/src/project/wallet/wallet.pml [hunk 3]
```diff
@@ -162,8 +162,8 @@ TransferService --|> WalletService
 
 TransferService --|> SDK
 
-SuggestionsService --|> SDK
-SuggestionsService --|> WalletSelectionService
+DebouncedSuggestionsService --|> SDK
+DebouncedSuggestionsService --|> SuggestionsService
```
Comment (marekkirejczyk) [inline docs/modeling/src/project/wallet/wallet.pml]: 👍

File: universal-login-commons/lib/core/models/WalletSuggestionAction.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
+export enum WalletSuggestionAction {
+  'create',
+  'connect',
+  'recover'
+}
+
+export const WALLET_SUGGESTION_ALL_ACTIONS: WalletSuggestionAction[] = [
```
Comment (marekkirejczyk) [inline universal-login-commons/lib/core/models/WalletSuggestionAction.ts]: WALLET_SUGGESTION_ACTIONS_ALL

File: universal-login-wallet/src/ui/react/Login/Login.tsx (hunk 1 above)
Comment (marekkirejczyk) [reply universal-login-wallet/src/ui/react/Login/Login.tsx]: Use constant

**Pull request merged**